dashboard/templates/dashboard/errors/404.html

```diff
@@ -0,0 +1,5 @@
+{% extends "dashboard/base.html" %}
+
+{% block content %}
+    <h2>Not Found</h2>
```

Comment:
Kinda nit-picky, but can we include "404" here too? I find sometimes I inspect page responses when they say not found to see if it's an actual 404, or if there's some other logic going on.